demo/android/STT/porcupine-stt-demo-app/src/main/res/mipmap-anydpi-v26/ic_launcher_round.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@color/ic_launcher_background"/>
```

Comment:
I noticed the logo says "Porcupine Stt-Android". Maybe let's just do "Porcupine STT" for now. If we do another STT demo, then maybe we can add the frameword to it.